Project-I-Social-Card/social-card/src/components/FooterComponents/Footer.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import "./Footer.css";
+
+function Footer() {
+  return (
```

Comment:
Right before the render - that's where the functions for the component should go. This is also where you would manage the state of the component!